lib/rubocop/cop/lint/redundant_safe_navigation.rb

```diff
@@ -7,6 +7,11 @@ module Lint
       # `instance_of?`, `kind_of?`, `is_a?`, `eql?`, `respond_to?`, and `equal?` methods
       # are checked by default. These are customizable with `AllowedMethods` option.
       #
+      # The `AllowedMethods` option specifies nil-safe methods,
+      # in other words, it is a method that is allowed to skip safe navigation.
+      # Note that the `AllowedMethod` option is not an option that specifies methods
+      # to suppressed (allowed) this cop offense.
```

Comment:
"methods for which to suppress (allow) this cop's check"

The above sounds better to me.